dms_field/static/src/js/base/dms_tree_controller.js

```diff
@@ -0,0 +1,585 @@
+                ]),
+                domain: args.storage.domain || [],
+                context: args.storage.context || session.user_context,
+                model: "dms.storage",
+                method: "search_read",
+            });
+        },
+        _loadDirectories: function (operator, value, args) {
+            return this._rpc({
+                model: "dms.directory",
+                method: "search_read_parents",
+                kwargs: {
+                    fields: _.union(args.directory.fields || [], [
+                        "permission_read",
+                        "permission_create",
+                        "permission_write",
+                        "permission_unlink",
+                        "count_directories",
+                        "count_files",
+                        "name",
+                        "parent_id",
+                        "__last_update",
+                    ]),
+                    domain: this._buildDMSDomain(
+                        [["storage_id", operator, value]],
+                        args.directory.domain,
+                        args.directory.autocompute_directory
+                    ),
+                    context: args.directory.context || session.user_context,
+                },
+            });
+        },
+        _loadDirectoriesSingle: function (storage_id, args) {
+            return this._loadDirectories("=", storage_id, args);
+        },
+        _loadSubdirectories: function (operator, value, args) {
+            return this._rpc({
+                model: "dms.directory",
+                method: "search_read",
+                fields: _.union(args.directory.fields || [], [
+                    "permission_read",
+                    "permission_create",
+                    "permission_write",
+                    "permission_unlink",
+                    "count_directories",
+                    "count_files",
+                    "name",
+                    "parent_id",
+                    "__last_update",
+                ]),
+                domain: this._buildDMSDomain(
+                    [["parent_id", operator, value]],
+                    args.directory.domain,
+                    false
+                ),
+                context: args.file.context || session.user_context,
+            });
+        },
+        _loadSubdirectoriesSingle: function (directory_id, args) {
+            return this._loadSubdirectories("=", directory_id, args);
+        },
+        _loadFiles: function (operator, value, args) {
+            return this._rpc({
+                model: "dms.file",
+                method: "search_read",
+                fields: _.union(args.file.fields || [], [
+                    "permission_read",
+                    "permission_create",
+                    "permission_write",
+                    "permission_unlink",
+                    "name",
+                    "res_mimetype",
```

Comment:
I believe the correct name now in 14.0 is mimetype.
https://github.com/OCA/dms/blob/46df04975e7df4976abf8003c25b8a0153674cd2/dms/migrations/14.0.1/pre-migration.py#L7

Reply:
Done! Thank you!